prow/config/config.go

```diff
@@ -264,6 +324,34 @@ func Load(path string) (*Config, error) {
 	return nc, nil
 }
 
+// mergeConfig merges two prow configs together
+// It will try to merge:
+//	- Presubmits
+//	- Postsubmits
+// 	- Periodics
+//	- PodPresets
+// It will reject and return an error if
+//	- multiple Prow config is defined
```

Comment:
Also check that presets, and jobs don't conflict.

Reply:
done